deflect/server/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+
+# Copyright (c) 2018, EPFL/Blue Brain Project
+#                     Raphael Dumusc <raphael.dumusc@epfl.ch>
+
+set(DEFLECTSERVER_PUBLIC_HEADERS
+  EventReceiver.h
+  Frame.h
+  Server.h
+)
+set(DEFLECTSERVER_HEADERS
+  FrameDispatcher.h
+  ServerWorker.h
+  ReceiveBuffer.h
+  SourceBuffer.h
+)
+set(DEFLECTSERVER_SOURCES
+  Frame.cpp
+  FrameDispatcher.cpp
+  Server.cpp
+  ServerWorker.cpp
+  ReceiveBuffer.cpp
+  SourceBuffer.cpp
+)
+
+set(DEFLECTSERVER_LINK_LIBRARIES
+  PUBLIC Deflect Qt5::Core Qt5::Network
```

Comment:
network is public?

Reply:
Because Server : public QTcpServer. It probably wouldn't be too hard to hide it in the future.